templates/redis_port.conf.erb

```diff
@@ -420,6 +420,54 @@ slowlog-log-slower-than <%= @redis_slowlog_log_slower_than %>
 # You can reclaim memory used by the slow log with SLOWLOG RESET.
 slowlog-max-len <%= @redis_slowlog_max_len %>
 
+<% if @redis_notify_keyspace_events %>
```

Comment:
I suspect this is going to introduce newlines in the config which, while not having any impact on the interpretation of the config file, will cause *some* changes and possibly cycle any redis servers that still have the service managed.